libraries/chain/account_evaluator.cpp

```diff
@@ -284,8 +284,8 @@ void_result account_update_evaluator::do_apply( const account_update_operation&
    {
       d.modify( acnt->statistics( d ), [&]( account_statistics_object& aso )
       {
-         if((o.new_options->votes != acnt->options.votes ||
-             o.new_options->voting_account != acnt->options.voting_account))
+         //if((o.new_options->votes != acnt->options.votes ||
+         //    o.new_options->voting_account != acnt->options.voting_account))
```

Comment:
please remove commented code directly.

Reply:
Why actually this is being removed ? This means all `account_update_operation` will update the last vote time and this will be considered activity(user will get rewards by updating his/her account while he/she can be not participating in the governance).

Reply:
Yeah, it was not supposed to be commented. I have start doing changes to update last_voting_time for each sub-period(especially repeating the same vote). I will update and request your review.